akka-actor/src/main/scala/akka/actor/AbstractActor.scala

```diff
@@ -93,6 +94,15 @@ object AbstractActor {
      */
     def getSystem(): ActorSystem
 
+    /**
+     * Returns the dispatcher (MessageDispatcher) that is used for this Actor.
+     * Importing this member will place an implicit ExecutionContext in scope.
+     *
+     * This method is thread-safe and can be called from other threads than the ordinary
+     * actor message processing thread, such as [[java.util.concurrent.CompletionStage]] and [[scala.concurrent.Future]] callbacks.
+     */
+    def getDispatcher(): ExecutionContextExecutor
+
     /**
```

Comment:
to make it getter complete we should also add `getProps` and `getReceiveTimeout`

Reply:
I added more, even they are a kind of repeating, but help a lot for Java developers.